re/aes.go

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Abstract Machines
+// SPDX-License-Identifier: Apache-2.0
+
+package re
+
+import (
+	"crypto/aes"
+	"crypto/cipher"
+	"encoding/hex"
+	"fmt"
+	"log"
+	"strings"
+)
+
+	/*
+	AES CBC-128 DECRYPTION requires 3 data fields
+	1. Key (16 bytes)
+	2. Initialization Vector (16 bytes). {Lua script generates this}
+	3. Encrypted Data (16 bytes or length be multiple a of 16) {Not the whole Telegram rather the encrypted part}
+
+	The encrypted data is divided into blocks of 16 bytes (128 bits) which then operated on with the IV and Key.   
+	*/
+	// Create a new AES cipher object with the given key and initialization vector.

```

Comment:
There is no need to indent. Also, switching to multiple single-line comments is more idiomatic in Go (for no obvious reason).